src/rewrite.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { RewriteRule, RewriteRules } from "./rewrite";
+
+describe("RewriteRule", () => {
```

Comment:
**Suggestion:** It might be worth adding a test for an expression similar to the one you use in the `similarity.spec.ts` (e.g. `jane.doe-some-test-6f82fbbe-d36a-4c5c-b47b-84100122fbbc@example.com`) to verify its output is expected and not just that it passes the similarity tests when running

Reply:
Done.